app-text/scdoc/scdoc-1.11.2.recipe

```diff
@@ -0,0 +1,41 @@
+SUMMARY="A small man page generator"
+DESCRIPTION="A simple man page generator for POSIX systems written in C99."
+HOMEPAGE="https://git.sr.ht/~sircmpwn/scdoc"
+COPYRIGHT="2017 Drew DeVault"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://git.sr.ht/~sircmpwn/scdoc/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="e9ff9981b5854301789a6778ee64ef1f6d1e5f4829a9dd3e58a9a63eacc2e6f0"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	scdoc$secondaryArchSuffix = $portVersion
+	cmd:scdoc = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	make
+}
+
+INSTALL()
+{
+	mkdir -p $manDir
+
+	make install PREFIX=$prefix
+	mv $prefix/share/man/* $manDir
+	rm -rf $prefix/share/
```

Comment:
From the Makefile:
```
PREFIX?=/usr/local
BINDIR?=$(PREFIX)/bin
MANDIR?=$(PREFIX)/share/man
PCDIR?=$(PREFIX)/share/pkgconfig
```

You can use these variables here, eg MANDIR=mandir ... saves the cleanup in the end

Reply:
thanks for the pointers! updated!